source/Octopus.Cli/Commands/Releases/ReleasePlanBuilder.cs

```diff
@@ -194,14 +198,24 @@ async Task<ReleasePlan> BuildReleaseFromDatabase(IOctopusAsyncRepository reposit
             return plan;
         }
 
-        static async Task<Dictionary<string, FeedResource>> LoadFeedsForSteps(IOctopusAsyncRepository repository, ProjectResource project, IEnumerable<ReleasePlanItem> steps)
+        async Task<Dictionary<string, FeedResource>> LoadFeedsForSteps(IOctopusAsyncRepository repository, ProjectResource project, IEnumerable<ReleasePlanItem> steps)
         {
             // PackageFeedId can be an id or a name
             var allRelevantFeedIdOrName = steps.Select(step => step.PackageFeedId).ToArray();
```

Comment:
I am surprised there isn't a DISTINCT going on here 🤔 Multiple actions could be referencing the same PackageFeedId. Maybe I'm missing something?

Reply:
Good catch. Looking at how this is used, it's just a lookup for the feeds, shouldn't need any duplicated.
It's not directly related but I've added a `Distinct()` while we're here.